android/src/main/java/com/mattermost/networkclient/APIClient.kt

```diff
@@ -124,4 +124,14 @@ class APIClientModule(reactContext: ReactApplicationContext) : ReactContextBaseJ
             promise.reject(e)
         }
     }
+
+    @ReactMethod
+    fun upload(baseUrl: String, endpoint: String?, fileUrl: String, taskId: String, options: ReadableMap, promise: Promise) {
+
+    }
+
+    @ReactMethod
+    fun cancelRequest(taskId: String) {
+
+    }
```

Comment:
are these supposed to be part of this PR?

Reply:
Yep; there's the upload PR incoming shortly - just git rebase conflict avoidance.

Reply:
If this is not part of the PR we should remove it, I rather keep the history clean than avoiding having to resolve conflicts when we submit PR's on top of each other (which we should also avoid).

Reply:
Easy, no probs - I'll remove.